src/bin/am/commands/start.rs

```diff
@@ -83,10 +84,110 @@ pub struct Arguments {
     ephemeral: bool,
 }
 
-pub async fn handle_command(mut args: Arguments, mp: MultiProgress) -> Result<()> {
-    if args.metrics_endpoints.is_empty() && args.enable_pushgateway {
-        let endpoint = interactive::user_input("Endpoint")?;
-        args.metrics_endpoints.push(endpoint_parser(&endpoint)?);
+#[derive(Debug, Clone)]
+struct InnerArguments {
```

Comment:
maybe we can name this `Arguments` an the ones passed from Cli `CliArguments`

Reply:
Renamed the structs as suggested